machine-learning/Dockerfile

```diff
@@ -25,4 +27,4 @@ ENV NODE_ENV=production \
 
 COPY --from=builder /opt/venv /opt/venv
 COPY app .
-ENTRYPOINT ["python", "-m", "app.main"]
+ENTRYPOINT ["tini", "--", "python", "-m", "app.main"]
```

Comment:
TBH idk why the python command is in the entry point. It should be in a CMD. Would it be possible to make it consistent with the other images?

Reply:
cc @mertalev ?

Reply:
I agree. Separating the python bits into `CMD` is a lot cleaner.

```Dockerfile
ENTRYPOINT [ "tini", "--" ]
CMD [ "python", "-m", "app.main" ]
```